xonsh/environ.py

```diff
@@ -926,6 +928,15 @@ def _current_job():
     return ''
 
 
+def env_name(pre_chars='(', post_chars=') '):
+    """Extract the current environment name from $VIRTUAL_ENV."""
+    env_path = builtins.__xonsh_env__.get('VIRTUAL_ENV', '')
```

Comment:
Do you want to please add an entry to `DEFAULT_VALUES` and then change this to `__xonsh_env__.get('VIRTUAL_ENV')`